tests/test_subquery_is_synced.py

```diff
@@ -0,0 +1,74 @@
+import requests
+import pytest
+
+subquery_urls = [
```

Comment:
That would be better to take subquery urls from config file, in order to not hardcode it.

example:
```python
import pytest
from typing import List
from scripts.utils.work_with_data import get_network_list

def get_unique_subquery_urls(network_list: List[dict]) -> List[str]:
    subquery_urls = set()
    for chain in network_list:
        external_api = chain.get('externalApi', {})
        for api_type, api_list in external_api.items():
            if isinstance(api_list, list):
                for api in api_list:
                    if api.get('type') == 'subquery':
                        subquery_urls.add(api['url'])
            elif isinstance(api_list, dict) and api_list.get('type') == 'subquery':
                subquery_urls.add(api_list['url'])
    return list(subquery_urls)

@pytest.fixture(scope="module")
def subquery_projects():
    network_file_path = "chains/v20/chains.json"
    network_list = get_network_list('/' + network_file_path)
    return get_unique_subquery_urls(network_list)
```

Reply:
this will return also subquery nodes that we host like : https://api.subquery.network/sq/nova-wallet/nova-wallet-polkadot is this ok?

Reply:
yeah, shouldn't be a problem. That may be a problem with multi-chain project, but this is stored in another file and we have statistics on our infrastructure and notifications